cfme/base/login.py

```diff
@@ -19,6 +19,7 @@ class BaseLoggedInPage(View):
     configuration_settings = Text('.//li[./a[@title="Configuration"]]')      # 5.11+
     settings = SettingsNavDropdown('.//li[./a[@id="dropdownMenu2"]]')
     navigation = VerticalNavigation('#maintab')
+    add_button = Text('//*[@id="main_div"]//div/a')
```

Comment:
There are many pages that use this class but not all of them have the `add_button` widget on them. Also, why not use the button widget for this? 

My thought is that this button should be on the `InfraProvidersView` page.  After doing the navigatation via `view = navgiate_to(appliance.server, "LoggedIn")`, you then can create the view and then and assert that the button is present. What do you think?